dask/base.py

```diff
@@ -458,9 +458,15 @@ def visualize(*args, **kwargs):
     optimize_graph : bool, optional
         If True, the graph is optimized before rendering.  Otherwise,
         the graph is displayed as is. Default is False.
-    color: {None, 'order'}, optional
+    color : {None, 'order'}, optional
         Options to color nodes.  Provide ``cmap=`` keyword for additional
         colormap
+    collapse : bool, optional
```

Comment:
I could quite guess what this would do based just on the name (but the images in the pull request made it clear). Thoughts on a more verbose name like `collapse_outputs` or `collapse_empty_outputs`?

Reply:
Hmm, I guess that doesn't make it clear that collapse also removes non-connected boxes?

Reply:
I think `collapse_outputs` seems good. I am not at all wedded to these names.